pom.xml

```diff
@@ -3371,5 +3371,44 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <id>partial-build</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-enforcer-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>enforce-environment-variable-is-set</id>
+                                <goals>
+                                    <goal>enforce</goal>
+                                </goals>
+                                <configuration>
+                                    <rules>
+                                        <requireEnvironmentVariable>
+                                            <variableName>JAMES_PARTIAL_BUILD_BASE_BRANCH</variableName>
```

Comment:
what about writing a nice message with <message> tag ?

Reply:
Here you are.